.buildkite/build-static.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+cd MUXSDKStats
+pod repo update
+pod deintegrate && pod install
+cd ..
+./update-release-xcframeworks-static.sh
+zip -ry MUXSDKStats.xcframework.zip XCFramework
```

Comment:
Let's add `static` to the resulting ZIP filename:
```suggestion
zip -ry MUXSDKStats-static.xcframework.zip XCFramework
```